main/zoe/api/zoe.md

```diff
@@ -0,0 +1,77 @@
+# Zoe
+
+Zoe is a long-lived and well-trusted contract that enforces offer safety for the contracts that use it. Zoe has a single `seatAssay` and a single `escrowReceiptAssay` for the entirety of its lifetime. By having a reference to Zoe, a user can get the `seatAssay` or `escrowReceiptAssay` and thus validate any `seat` or `escrowReceipt` they receive from someone else.
+
+### <span style="color:red">Zoe and ZoeGoverningContractFacet have some similar methods so those examples are the same. Those should be double-checked to make sure they are accurate in both locations. Methods: getSeatAssay, getEscrowReceiptAssay.</span>
+
+
+## zoe.getSeatAssay()
```

Comment:
This should be "getInviteAssay()" (my fault for not updating it) but looks good otherwise